src/Locales/es/Pode.psd1

```diff
@@ -0,0 +1,287 @@
+    nonEmptyScriptBlockRequiredForCustomLoggingExceptionMessage       = 'Se requiere un ScriptBlock no vacío para el método de salida de registro personalizado.'
+    encodingAttributeOnlyAppliesToMultipartExceptionMessage           = 'El atributo de codificación solo se aplica a cuerpos de solicitud multipart y application/x-www-form-urlencoded.'
+    suppliedDateBeforeScheduleStartTimeExceptionMessage               = 'La fecha proporcionada es anterior a la hora de inicio del horario en {0}'
+    unlockSecretRequiredExceptionMessage                              = "Se requiere una propiedad 'UnlockSecret' al usar Microsoft.PowerShell.SecretStore"
+    noLogicPassedForMethodRouteExceptionMessage                       = '[{0}] {1}: No se pasó lógica.'
+    bodyParserAlreadyDefinedForContentTypeExceptionMessage            = 'Un analizador de cuerpo ya está definido para el tipo de contenido {0}.'
+    invalidJwtSuppliedExceptionMessage                                = 'JWT proporcionado no válido.'
+    sessionsRequiredForFlashMessagesExceptionMessage                  = 'Se requieren sesiones para usar mensajes Flash.'
+    semaphoreAlreadyExistsExceptionMessage                            = 'Ya existe un semáforo con el siguiente nombre: {0}'
+    invalidJwtHeaderAlgorithmSuppliedExceptionMessage                 = 'Algoritmo del encabezado JWT proporcionado no válido.'
+    oauth2ProviderDoesNotSupportPasswordGrantTypeExceptionMessage     = "El proveedor de OAuth2 no admite el tipo de concesión 'password' requerido al usar un InnerScheme."
+    invalidAliasFoundExceptionMessage                                 = 'Se encontró un alias {0} no válido: {1}'
+    scheduleDoesNotExistExceptionMessage                              = "El horario '{0}' no existe."
+    accessMethodNotExistExceptionMessage                              = 'El método de acceso no existe: {0}'
+    oauth2ProviderDoesNotSupportCodeResponseTypeExceptionMessage      = "El proveedor de OAuth2 no admite el tipo de respuesta 'code'."
+    untestedPowerShellVersionWarningMessage                           = '[ADVERTENCIA] Pode {0} no se ha probado en PowerShell {1}, ya que no estaba disponible cuando se lanzó Pode.'
+    secretVaultAlreadyRegisteredAutoImportExceptionMessage            = "Ya se ha registrado un Bóveda Secreta con el nombre '{0}' al importar automáticamente Bóvedas Secretas."
+    schemeRequiresValidScriptBlockExceptionMessage                    = "El esquema proporcionado para el validador de autenticación '{0}' requiere un ScriptBlock válido."
+    serverLoopingMessage                                              = 'Bucle del servidor cada {0} segundos'
+    certificateThumbprintsNameSupportedOnWindowsExceptionMessage      = 'Las huellas digitales/nombres de certificados solo son compatibles con Windows.'
+    sseConnectionNameRequiredExceptionMessage                         = "Se requiere un nombre de conexión SSE, ya sea de -Name o $`$WebEvent.Sse.Name"
```

Comment:
Typo in the escaping of "$" for "$WebEvent", there's an erroneous "$" in front of the backtick